.env-example

```diff
@@ -34,7 +34,8 @@ DEFAULT_SERVER_EMAIL='mail <no-reply@site.me>'
 DJANGO_SENTRY_DSN=''
 DJANGO_SENTRY_LOG_LEVEL=20
 
-JWT_SECRET_KEY='SECRET_KEY'=your jwt secret key
+JWT_SECRET_KEY=your jwt secret key
+swagger_url=the url to the server running the application
```

Comment:
Hello @IssaIan ,
Change the `swagger_url` environment variable to `SWAGGER_URL`